.github/workflows/analyze.yml

```diff
@@ -69,14 +69,21 @@ jobs:
       # Either of these arguments can be changed or removed by editing the `nextBundleAnalysis`
       # entry in your package.json file.
       - name: Compare with base branch bundle
-        if: success() && github.event.number
+        if: success()
         run: ls -laR .next/analyze/base && npx -p nextjs-bundle-analysis compare
 
       - name: Upload analysis comment
         uses: actions/upload-artifact@v2
         with:
           name: analysis_comment.txt
           path: .next/analyze/__bundle_analysis_comment.txt
+          
+  number:
```

Comment:
We shouldn't upload the pr_number when we push directly. So I split a new job and add a condition in order to run it on pull_request only.